Environment_Engine/Convert/Line.cs

```diff
@@ -0,0 +1,87 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2019, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Geometry;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using BH.oM.Environment;
+using BH.oM.Environment.Elements;
+using BH.Engine.Geometry;
+
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
+
+namespace BH.Engine.Environment
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /****          public Methods                   ****/
+        /***************************************************/
+
+        [Description("BH.Engine.Environment.Convert.ToLines => Returns a collection of Lines representing an Environment Edge")]
+        [Input("edge", "An Environment Edge object")]
+        [Output("lines", "A collection of BHoM Geometry Line")]
+        public static List<Line> ToLines(this Edge edge)
```

Comment:
Similar to the `ToEdge()` method above, I would not see this as a convert but as a query.

Almost philosophical, but I see this as getting a piece of geometry out from an object, rather than turning an object into a piece of geometry.